### DISCUSSION TITLE
Combined package WeConnect-MQTT / VWsFriends?

### DISCUSSION THREAD

**Discussion by wizzywik** [tillsteinbach/WeConnect-mqtt#71] [Ideas]
Hi Till,
Is it conceivable that there will be a combined package for both monitoring the car (like VWsFriends-Grafana) and controlling the car (start/stop charging as in the WeConnect API)?
My situation/wish in more detail:
- love data to understand what is happening with my car in different situations (VWsFriends with Grafana will do that job, I love the package)
- want to control charging from my home-automation (Loxone) to optimise the available solar-power (smart-charging)
- I'm not a programmer, but managed to install all your packages in Docker, can talk MQTT, understand databases and a bit of scripting and interfaces
Thnx for your thoughts!

**Comment by tillsteinbach** [comment]
May you can elaborate a bit more what your goal is. It is not a problem to run VWsFriend and WeConnect-MQTT at the same time. You can even put it in the same docker-compose file if you like. The only downside of this approach is that there is a bit of overhead because both tools will query the same data from the servers.

**Comment by wizzywik** [comment]
Well that's exactly my goal, I don't want to stress VW servers, but love to have near-realtime data. If the connection to VW can be combined for multiple purposes, that would lower the load. As an alternative: is it possible to add a very simple API to VWsFriends for just start/stop charging? (Or is that already possible?)

**Reply by tillsteinbach** [reply]
No it is not yet possible (at least not very simple) There is the HomeKit API, but I doubt it is easy to send the commands there.
I might add a simple Web Form later, but it does not have high priority right now.

I could also think of somehow have a good abstraction layer for the MQTT part and then having WeConnect-MQTT stand alone and also the same functionality optionally in VWsFriend. But that is also not too high in the priorities right now.
